src/crimson/osd/ops_executer.cc

```diff
@@ -475,6 +475,19 @@ OpsExecuter::call_errorator::future<> OpsExecuter::do_assert_ver(
 
 OpsExecuter::interruptible_errorated_future<OpsExecuter::osd_op_errorator>
 OpsExecuter::execute_op(OSDOp& osd_op)
+{
+  return (osd_op.op.flags & CEPH_OSD_OP_FLAG_FAILOK) ?
+    do_execute_op(osd_op).handle_error_interruptible(
+      crimson::ct_error::eagain::pass_further{},
+      crimson::ct_error::einprogress::pass_further{},
+      osd_op_errorator::all_same_way([] {
```

Comment:
How about using `osd_op_errorator::discard_all{}` instead of `all_same_way`?

Reply:
> How about using `osd_op_errorator::discard_all{}` instead of `all_same_way`?

there is build error, seems discard_call return seastar:future, not consistent with do_execute_op, I can't make a type conversion for discall_all{}.  Any suggestion?